README.md

```diff
@@ -535,14 +543,54 @@ Changes the editor interface of given field's ID.
 - **`ampm : string`** _(only for fields of type datePicker)_ – Specifies which type of clock to use. Must be one of the strings “12” or “24” (default).
 - **`bulkEditing : boolean`** _(only for fields of type Array)_ – Specifies whether bulk editing of linked entries is possible.
 
-#### `resetEditorInterface (fieldId)` : void
+#### `resetFieldControl (fieldId)` : void
 
 **`fieldId : string`** – The ID of the field.
 
-#### `copyEditorInterface (sourceFieldId, destinationFieldId)` : void
+#### `copyFieldControl (sourceFieldId, destinationFieldId)` : void
+
+**`sourceFieldId : string`** – The ID of the field to copy the control setting from.
+**`destinationFieldId : string`** – The ID of the field to apply the copied control setting to.
+
+#### `addSidebarWidget (widgetId, widgetNamespace[, insertBeforeWidgetId, settings])` : void
```

Comment:
I'd swap `insertBeforeWidgetId` and `settings`. It would seem more aligned with other methods and won't require to set `insertBeforeWidgetId` to null if you want to skip it.